Adding Barrel plugin

wrexroad opened the pull request:
Created a first version of the BARREL loading function and config.

jameswilburlewis commented:
Hi Warren,

Thanks for contributing!  I've merged your commits into the master pyspedas branch, with one change:  The default date range for the load routines doesn't have any data available at SPDF, so I changed it from ['2017-01-17','2017-01-19'] to ['2017-01-28','2017-01-29'].  I've also added your test suite to our Github action that runs after pushing anything to the master branch.

wrexroad commented:
Thanks for catching that! I will be adding more comprehensive tests and
issuing a pull request for pyspedas_examples soon.

-Warren

On Thu, Aug 24, 2023 at 10:29 AM Jim Lewis ***@***.***> wrote:

> Hi Warren,
>
> Thanks for contributing! I've merged your commits into the master pyspedas
> branch, with one change: The default date range for the load routines
> doesn't have any data available at SPDF, so I changed it from
> ['2017-01-17','2017-01-19'] to ['2017-01-28','2017-01-29']. I've also added
> your test suite to our Github action that runs after pushing anything to
> the master branch.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/spedas/pyspedas/pull/577#issuecomment-1692134169>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AARI2IAGSVOKILHDFPRQBC3XW6FOZANCNFSM6AAAAAA34RQUCI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
